Caboodle/Geolocation/Geolocation.ios.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using CoreLocation;
+using Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Geolocation
+    {
+        internal static bool IsSupported
+            => CLLocationManager.LocationServicesEnabled;
+
+        static async Task<Location> PlatformLastKnownLocationAsync()
+        {
+            await Permissions.RequireAsync(PermissionType.LocationWhenInUse).ConfigureAwait(false);
+
+            var manager = new CLLocationManager();
+            var location = manager.Location;
+
+            return new Location
+            {
+                Latitude = location.Coordinate.Latitude,
+                Longitude = location.Coordinate.Longitude,
+                Accuracy = location.HorizontalAccuracy,
+                TimestampUtc = ToDate(location.Timestamp)
+            };
+        }
+
+        static async Task<Location> PlatformLocationAsync(GeolocationRequest request, CancellationToken cancellationToken)
+        {
+            await Permissions.RequireAsync(PermissionType.LocationWhenInUse).ConfigureAwait(false);
+
+            var manager = new CLLocationManager();
```

Comment:
This location manager is different to the one used in permissions. Need to review if this gets GC'ed unexpectedly.